AutomatedTesting/Assets/AutomatedTestScripts/screenshot.sa.lua

```diff
@@ -9,12 +9,12 @@
 --
 ----------------------------------------------------------------------------------------------------
 
-g_screenshotOutputFolder = GetScreenshotOutputPath(true)
-Print('Saving screenshots to ' .. NormalizePath(g_screenshotOutputFolder))
+g_screenshotOutputFolder = '@user@/ScriptAutomation/Screenshots/'
+Print('Saving screenshots to ' .. ResolvePath(g_screenshotOutputFolder))
```

Comment:
Do these lines do anything? CaptureScreenshot builds it's own capture path now.

Reply:
It just prints out the message to the console, which doesn't happen when a capture is taken.
I can make this a script call so that things can become neater.

Reply:
It just prints out the message to the console, which doesn't happen when a capture is taken.
I can make this a script call so that things can become neater.